lua/opencode/ui/completion/engines/blink_cmp.lua

```diff
@@ -55,12 +54,20 @@ function Source:get_completions(ctx, callback)
     for i, item in ipairs(source_items) do
       table.insert(items, {
         label = item.label,
-        kind = item.kind == 'file' and 17 or item.kind == 'folder' and 19 or 1, -- 17: File, 19: Folder, 1: Text
+        kind = item.kind,
+        kind_icon = item.kind_icon,
```

Comment:
If we set kind_hl, we can differentiate enabled vs not:

<img width="315" height="168" alt="Screenshot 2025-10-30 at 16 36 25" src="https://github.com/user-attachments/assets/65f6319d-048f-4636-9745-c44ad8571bd5" />

For that, I just added this:

```lua
    kind_hl = not icon and available and 'WarningMsg' or nil,
```

to:
https://github.com/sudo-tee/opencode.nvim/blob/ff0ba7f5250cd37e1d8a3d2c8698d247f2de7ba0/lua/opencode/ui/completion/context.lua#L24-L28

And then just `kind_hl = item.kind_hl` in code this comment is linked to.

Reply:
Oh I like that.  thanks